template_app_config_local.sh

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env bash
+
+#
+# Script for templating the jinja2 app-config.local.yaml file.
+
+set -o errexit
+
+log_message() {
+    VERBOSITY_LEVEL=$1
+    MESSAGE="${@:2}"
+    if [ "${LOGGING_VERBOSITY}" -ge "${VERBOSITY_LEVEL}" ]; then
+        echo -e "${MESSAGE}"
+    fi
+}
+
+log_message_nonl() {
+    VERBOSITY_LEVEL=$1
+    MESSAGE="${@:2}"
+    if [ "${LOGGING_VERBOSITY}" -ge "${VERBOSITY_LEVEL}" ]; then
+        echo -ne "${MESSAGE}"
+    fi
+}
+
+# Check the system requirements
+check_requirements() {
+    if ! command -v jinja2 &> /dev/null
+    then
+        log_message 0 "ERROR: jinja2 not installed! Install it using:"
+        log_message 0 "  sudo dnf install python3-jinja2-cli"
```

Comment:
This command only works on Linux RHEL OS and will fail on MacOS !
```
./template_app_config_local.sh
ERROR: jinja2 not installed! Install it using:
  sudo dnf install python3-jinja2-cli
```

Reply:
Is it working already?

Reply:
Yes.

Reply:
Changed the error message so it points to the project URL.